firmware/README.md

```diff
@@ -4,18 +4,26 @@ CYW43xx WiFi SoC firmware
 This directory contains firmware patch blobs that need to be downloaded on to the
 CYW43xx SoC in order for it to function correctly.
 
-The firmware is padded to 512 bytes and then the CLM appended to that, to create the
-combined binary file.
+The Wifi firmware and CLM data is padded to 512 bytes. Then the Wifi firmware, CLM data and BT firmware are all appended together,
+to create the combined binary file.
```

Comment:
The BT firmware is no longer appended, so maybe say: "The WiFi firmware is padded to 512 bytes and then the CLM appended to that, to create the combined WiFi and CLM binary data."

Reply:
I wrote this - ok? "The WiFi firmware is padded to 512 bytes and then the CLM appended to that to create the combined binary file."

Reply:
looks good